lte/gateway/docker/services/pkt_tester/Dockerfile

```diff
@@ -1,12 +0,0 @@
-FROM ubuntu:20.04 as pkt_tester
```

Comment:
@vktng , can we remove this directory too since this is the only file under  `lte/gateway/docker/services/pkt_tester`

Reply:
Do you mean `lte/gateway/docker/services/pkt_tester`? Git does not track empty directories, so it should be removed implicitly.

Reply:
Would it also delete it from my working directory if I pull the changes after this PR is merged?

Reply:
hmm just tested - for me it's removed when I switch branches. Does not coincide with what I remembered regarding git behavior :) I was under the impression that you have to clean-up directories that become empty yourself ... by e.g., `git clean -fd` (add `-n` for a dry run to see what would be removed).

Reply:
That's neat! Merging this PR.